recipes/virsorter/meta.yaml

```diff
@@ -10,24 +10,25 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0      # must start from 0
+  number: 1      # must start from 0
   entry_points:
     - virsorter=virsorter.virsorter:cli
   noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.6,<=3.10
```

Comment:
why is the upper pin here needed?

Reply:
To avoid the issues caused by changes in new versions, eg. situation like this  update.

Reply:
@bgruening Could you approve this? or any other comments?  Thanks!